src/mainboard/pcengines/apu1/platform_cfg.h

```diff
@@ -213,4 +213,17 @@
  */
 #define FADT_PM_PROFILE 1
 
+/**
+ * @def USB_RX_MODE
+ *    0x00 - leave Cg2Pll voltage at default value (1.222V)
+ *    0x01 - lower Cg2Pll voltage to 1.1V
+ *
+ * Workaround for reset issues via outb(0x6, 0xcf9).
+ * For details check:
+ * AMD SB800 Family Product Errata,
+ *     Section 15. USB Resets Asynchronously With Port CF9h Hard Reset
```

Comment:
@piotr-kleins have you tried running normal regression on apu1?

Reply:
Did you experience the issues as described in this errata?

Reply:
@miczyg1 Errata says: The system may hang on initialization after the reset and it happens on apu1, so yes I did. Implementing this fix resolved the issue and no more hangs occurred.
I haven't run regression tests yet, but it will be done soon.